portfolio/src/main/webapp/style.css

```diff
@@ -49,6 +58,11 @@ p{
     font-size: large;
 }
 
+hr{
```

Comment:
Where you planning on adding a specific ID to each `<hr>` element?

Reply:
Sorry I let laziness take over and I made it so the hr tags were the same. I will keep the tips you provided in mind for future pull requests, thank you for approving the request